### PR TITLE
Redirect all output from the wrapper to standard error. Removed all r…

### PR DESCRIPTION
**Pull request by vinjana**
…eopening redirections `>` to fix bug https://github.com/eilslabs/Roddy/issues/129.

### CODE REVIEW

File: resources/analysisTools/roddyTools/wrapInScript.sh [hunk 1]
```diff
@@ -22,6 +24,66 @@ fi
 # Cluster options (like i.e. PBS ) have to be parsed and set before job submission!
 # They will be ignored after the script is wrapped.
 
+
```
Comment (dankwart-de) [inline resources/analysisTools/roddyTools/wrapInScript.sh:27]: That's all in the wrapper script now, maybe you could move it to a different file?

Comment (dankwart-de) [reply resources/analysisTools/roddyTools/wrapInScript.sh:27]: Aren't these functions already available in other scripts?

**Review by dankwart-de — APPROVED** [review]

File: resources/analysisTools/roddyTools/wrapInScript.sh (hunk 1 above)
Comment (vinjana) [reply resources/analysisTools/roddyTools/wrapInScript.sh:27]: Yes, they are from the workflow library. We could move them into a general workflow library in the default plugin and expose this into the other plugins. Here the code is used in a very early stage of the script, where nothing more is known, but that the wrapper was loaded. Therefore I tended to put the code here. But if we can be sure that a DefaultPlugin workflowLib.sh is equally available, we can source it here.

Comment (dankwart-de) [reply resources/analysisTools/roddyTools/wrapInScript.sh:27]: If you put the script in the same directory as the wrapper, you can assume that it is this way. Only the name of the script will not be passed by Roddy. But I think in this case it is legitimate.

Comment (vinjana) [reply resources/analysisTools/roddyTools/wrapInScript.sh:27]: You mean, we don't define a tool in the DefaultPlugin XML?

**Comment by vinjana** [comment]
I could move the code out of the wrap in script, but am not sure, how necessary this would be now. I think we should postpone this until we really need the code somewhere else (in the core or default plugin; in the bioinformatics plugins it's fine if there is an independently developing version, I think). Do you agree @dankwart-de ?

**Pull request merged**